lib/firecord/credentials.rb

```diff
@@ -0,0 +1,40 @@
+require 'jwt'
+
+module Firecord
+  class Credentials
+    def initialize
+      @credentials = JSON.parse(File.read("./credentials.json"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)